### PR TITLE
Added `add-all` / `remove-all` / `show` commands to the role management commands

### LINKED ISSUE #26: Role management command to add `add-all` / `remove-all` / `show`
`add-all` - Add all owned tracked roles.
`remove-all` -  Remove all tracked roles.
`show` - Shows all the owned tracked roles.

### PR DESCRIPTION
**Pull request by Nokoyohei**
Closes #26

### CODE REVIEW

File: Eevee.Sleep.Bot/Utils/DiscordMessageMaker.cs [hunk 1]
```diff
@@ -194,6 +194,48 @@ ButtonId buttonId
...
+                    "(N/A)" :
+                    string.Join(" / ", currentRoleIds.Select(x => MentionUtils.MentionRole(x)))
+            )
+            .WithCurrentTimestamp()
+            .Build();
+    }
+
+    public static Embed MakeShowRoleResult(IUser user, ulong[] roleIds) {
+        return new EmbedBuilder()
+            .WithColor(Colors.Success)
```
Comment (RaenonX) [inline Eevee.Sleep.Bot/Utils/DiscordMessageMaker.cs:230]: Use `Colors.Info` instead.

File: Eevee.Sleep.Bot/Utils/DiscordMessageMaker.cs [hunk 2]
```diff
@@ -194,6 +194,48 @@ ButtonId buttonId
...
+            .WithAuthor(user)
+            .WithTitle("Your owned roles")
+            .AddField(
+                name: "Roles",
+                value: roleIds.Length == 0 ?
```
Comment (RaenonX) [inline Eevee.Sleep.Bot/Utils/DiscordMessageMaker.cs]: Refactor all `string.Join()` calls to `DiscordExtension.MentionAllRoles()` and call it from there.

File: Eevee.Sleep.Bot/Modules/SlashCommands/RoleManagementSlashModule.cs [hunk 3]
```diff
@@ -83,6 +83,72 @@ public class RoleManagementSlashModule : InteractionModuleBase<SocketInteraction
...
+                color: Colors.Danger
+            ),
+            ephemeral: true
+        );
+    }
+
+    [SlashCommand("show", "Shows all the owned tracked roles.")]
+    [UsedImplicitly]
+    public Task ShowRoleAsync() {
+        var user =
```
Comment (RaenonX) [inline Eevee.Sleep.Bot/Modules/SlashCommands/RoleManagementSlashModule.cs]: Refactor all `Context.User as SocketGuildUser ?? ...` calls to `DiscordExtension.AsGuildUser()` then call it like `Context.User.AsGuildUser()`.

**Review by RaenonX — CHANGES_REQUESTED** [review]

**Comment by RaenonX** [comment]
Also just make changes based on 1 commit (the current one)

File: Eevee.Sleep.Bot/Utils/DiscordMessageMaker.cs (hunk 2 above)
Comment (RaenonX) [reply Eevee.Sleep.Bot/Utils/DiscordMessageMaker.cs]: And I think all `string.Join("\n")` can be further refactored to `StringExtension.mergeLines()` as well

**Comment by Nokoyohei** [comment]
I don't know if it matches your image, but I tried to fix it.

**Review by RaenonX — CHANGES_REQUESTED** [review]
Yeah exactly these!

Actually for `MergeLines()` and `AsGuildUser()` refactoring, make them 2 separate commits (1 for `MergeLines()`; 1 for `AsGuildUser()`. Then, for each commit, change all original `string.Join("\n")` usages and `as SocketGuildUser ??` usages to use these 2 implementations.

**Review by RaenonX — CHANGES_REQUESTED** [review]
Also refactor all the other `string.Join(" / ",` calls to use `MergeLines()` and merge that change into their corresponding commit.

This should be the last change needed because I am not seeing socket user casting anywhere!

**Comment by Nokoyohei** [comment]
Ok, probably all fixed.

**Review by RaenonX — APPROVED** [review]

**Comment by RaenonX** [comment]
I added a few more commits myself, but the rest are good!

**Pull request merged**